plugins/module_utils/database.py

```diff
@@ -140,3 +157,52 @@ def mysql_quote_identifier(identifier, id_type):
             special_cased_fragments.append(fragment)
 
     return '.'.join(special_cased_fragments)
+
+
+def is_input_dangerous(string):
+    """Check if the passed string is potentially dangerous.
+    Can be used to prevent SQL injections.
+
+    Note: use this function only when you can't use
+      psycopg2's cursor.execute method parametrized
+      (typically with DDL queries).
+    """
+    if not string:
+        return False
+
+    if re.search(PATTERN_1, string):
+        return True
+
+    if re.search(PATTERN_2, string):
+        return True
+
+    if re.search(PATTERN_3, string):
+        return True
+
+    return False
```

Comment:
Not sure if it matters but this could be simplified to:
```suggestion
    return (
        re.search(PATTERN_1, string)
        or re.search(PATTERN_2, string)
        or re.search(PATTERN_3, string)
    )
```

Reply:
Looks good! thanks! I committed you code, will see

Reply:
it works, thanks!
but i changed it a bit, otherwise units failed while i put return False explicitly. I think it returns None if searches don't find anything